app/models/spree/relation_type.rb

```diff
@@ -5,4 +5,5 @@ class Spree::RelationType < ApplicationRecord
 
   validates :name, :applies_from, :applies_to, presence: true
   validates :name, uniqueness: { case_sensitive: false }
+  validates :bidirectional, exclusion: { in: [true] }, unless: Proc.new { |rt| rt.applies_from == rt.applies_to }
```

Comment:
This is a bit complex to understand and it will also generate a very obscure validation message. 😅 Isn't it better to do it with a custom validation method?

Reply:
I've created a custom validation method and also a custom error message, let me know what your thoughts are.